api/users/serializers.py

```diff
@@ -444,6 +449,20 @@ def verify_two_factor(self, instance, value, two_factor_addon):
             raise exceptions.PermissionDenied(detail='The two-factor verification code you provided is invalid.')
         two_factor_addon.save()
 
+    def request_deactivation(self, instance, requested_deactivation):
+        if instance.requested_deactivation != requested_deactivation:
+            if requested_deactivation:
```

Comment:
Might want to use `is_truthy(requested_deactivation)` to make it a bit easier on the frontend

Reply:
I made the change, but then thought more about it, because I've never done this in v2. I looked at the [BooleanField](https://github.com/encode/django-rest-framework/blob/f539c0dbd13691a32637ff67bc384c8fcef2b636/rest_framework/fields.py#L656) definition - it's pretty permissive about what it excepts as True/False, and if it's any of the "truthy" values, it goes ahead and converts it to True.  So I think the logic in `is_truthy` is already covered in the BooleanField!